packages/eas-cli/src/build/runBuildAndSubmit.ts

```diff
@@ -180,13 +180,23 @@ export async function runBuildAndSubmitAsync(
     buildCtxByPlatform[platform] = buildCtx;
   }
 
-  if (flags.localBuildOptions.localBuildMode) {
+  if (flags.localBuildOptions.localBuildMode === LocalBuildMode.LOCAL_BUILD_PLUGIN) {
     return;
   }
 
-  Log.newLine();
-  printLogsUrls(startedBuilds.map(startedBuild => startedBuild.build));
-  Log.newLine();
+  if (flags.localBuildOptions.localBuildMode === LocalBuildMode.INTERNAL) {
+    const startedBuild = await BuildQuery.byIdAsync(
+      graphqlClient,
+      nullthrows(process.env.EAS_BUILD_ID, 'EAS_BUILD_ID is not defined')
+    );
+    startedBuilds.push({ build: startedBuild, buildProfile: buildProfiles[0] });
```

Comment:
Is `buildProfiles[0]` safe? Should we use `startedBuild.buildProfile.profile` instead?

Reply:
It's safe, this array can have a max length of 2 (one build profile for iOS and one for Android) and a min length of 1. In this case, it will always have length 1 for the `internal` command because we are concerned about the single build for a given platform with a given ID.

`startedBuild.buildProfile.profile` - it won't work, it's the data from GQL, and `startedBuild.buildProfile` is a string. We don't store build profiles in WWW.